build.gradle.kts

```diff
@@ -21,8 +21,8 @@ dependencies {
   implementation("org.springdoc:springdoc-openapi-ui:1.6.8")
   implementation("org.springdoc:springdoc-openapi-kotlin:1.6.8")
```

Comment:
can't update springdoc due to https://github.com/springdoc/springdoc-openapi/issues/1734